aten/src/ATen/native/native_functions.yaml

```diff
@@ -1210,6 +1210,10 @@
   supports_named_tensor: True
   variants: function, method
 
+- func: expm1.dtype(Tensor self, ScalarType dtype) -> Tensor
```

Comment:
does this work as an additional optional argument on the existing signature? e.g. `func: expm1(Tensor self, *, ScalarType? dtype=None)`

Reply:
Unfortunately, no! I have been trying this for a while now, but it always hits a roadblock, and eventually we'll have to change the functions name (not overloading) and add more helper functions. This seems messy to me at first.

Reply:
As-written this allows calls like `expm1(self, dtype)` (positionally) which wouldn't be allowed by the equivalent numpy api.

Reply:
This can be made keyword only argument (by putting the argument after `*` in `native_functions.yaml`) without needing an optional argument on the existing signature. I'll make a commit with a keyword-only approach and ping once done. 

I'm still trying with optional argument, but it's gonna change a lot of signatures in the `yaml` file. I shall post the exact errors I get once am done with the possible combinations of modifications I've in my mind.